### PR TITLE
Rebrand Onegini -> OneWelcome

### PR DESCRIPTION
**Pull request by osvso**

### CODE REVIEW

File: docs/dabp/release-notes/index.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Release notes
 
-This section contains release notes for Onegini DABP.
+This section contains release notes for OneWelcome DABP.
 
-In the release notes we mention new features and bug fixes. If anything is unclear, feel free to contact [Onegini Support](https://support.onegini.com).
+In the release notes we mention new features and bug fixes. If anything is unclear, feel free to contact [OneWelcome Support](https://support.onegini.com).
```
Comment (rvroon) [inline docs/dabp/release-notes/index.md:5]: Also change this domain

File: docs/identity-providers.md [hunk 2]
```diff
@@ -9,19 +9,19 @@ An Identity Provider is a system that creates, maintains, and manages identity i
...
-- You can add connections to social Identity Providers like Google, Twitter, or Facebook. It allows your users to sign in to your app using credentials from their social accounts. After users authenticate, you sync certain attributes from their social account to an Identity in your Onegini user store, while continuing to use that Identity Provider for user authentication. This eliminates the need to store an additional username and password for that user.
+- You can add connections to social Identity Providers like Google, Twitter, or Facebook. It allows your users to sign in to your app using credentials from their social accounts. After users authenticate, you sync certain attributes from their social account to an Identity in your OneWelcome user store, while continuing to use that Identity Provider for user authentication. This eliminates the need to store an additional username and password for that user.
 - You can add connections to predefined federated Identity Providers that are active in our primary markets. The following link contains a [complete list of the Identity Providers that we support](https://www.onegini.com/knowlegde/apis-integrations#identity_proofing).
```
Comment (rvroon) [inline docs/identity-providers.md:13]: Maybe we should remove this line, as the page no longer exists.

**Review by rvroon — CHANGES_REQUESTED** [review]

**Review by rvroon — APPROVED** [review]

**Review by czarnecki-piotr — APPROVED** [review]

File: docs/products/access/topics/tokens/access-token.md [hunk 3]
```diff
@@ -47,7 +47,7 @@ sections are displayed in the example below.
 {
   "ver": 1,
   "jti": "AT.d405c8b0-2afc-4720-a567-e890fecd28b2",
-  "iss": "https://token-server.onegini.com/oauth",
+  "iss": "https://token-server.onewelcome.com/oauth",
```
Comment (rvroon) [inline docs/products/access/topics/tokens/access-token.md:50]:
```suggestion
  "iss": "https://access.onewelcome.io/oauth",
```

File: docs/products/access/topics/tokens/id-token.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ In this section you can see an example of an ID Token. A JWT contains three sect
   "sub": "1c0e2c84-b05f-4c23-9175-c238f70901be",
   "email_verified": true,
   "gender": "male",
-  "iss": "https://token-server.onegini.com/oauth",
+  "iss": "https://token-server.onewelcome.com/oauth",
```
Comment (rvroon) [inline docs/products/access/topics/tokens/id-token.md:28]:
```suggestion
  "iss": "https://tenant.onewelcome.io/oauth",
```

**Review by czarnecki-piotr — APPROVED** [review]

**Pull request merged**